rules/embeds/instagram.js

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+/**
+ * @module rules/embeds/instagram
+ */
+
+const safeObject = require('../../util/safe_object');
+
+/**
+ * Add support for [Instagram](https://instagram.com/) embeds in Markdown, as block syntax.
+ *
+ * The basic syntax is `[instagram <post>]`. E.g. `[instagram https://www.instagram.com/p/CkQuv3_LRgS]`.
+ * After the post, assorted space-separated flags can be added (in any combination/order):
+ *
+ * - Add `caption` to include caption under the post.
+ * - Add `left`, `center`, or `right` to set the alignment of the embed (default is `left`).
+ * - Add any set of digits to set the width of the embed (in pixels, between 326 and 550, default is 326 not enforced here, but provided by Instagram's embed.js).
```

Comment:
```suggestion
 * - Add any set of digits to set the width of the embed (in pixels, between 326 and 550, default is 326 as set by Instagram's embed.js).
```
Just cleaning up the wording here (please do the same to the README)